packages/iTwinUI-react/src/core/Slider/Thumb.tsx

```diff
@@ -144,7 +153,12 @@ export const Thumb = (props: ThumbProps) => {
         {...rest}
         data-index={index}
         ref={thumbRef}
-        style={{ ...style, left: `${leftPercent}%` }}
+        style={{
+          ...style,
+          ...(orientation === 'horizontal'
+            ? { left: `${leftPercent}%` }
```

Comment:
Rename `leftPercent` to something more generic like `positionPercent` or something better.

Reply:
I renamed `leftPercent` to `lowPercent` and `rightPercent` to `highPercent`